test/loader/loader.test.js

```diff
@@ -46,5 +46,10 @@ describe('loader command', () => {
         files.forEach((file) => {
             expect(existsSync(join(__dirname, `${loaderName}/${file}`))).toBeTruthy();
         });
+
+        //check if the the generated plugin works successfully
+        const path = resolve(__dirname, './test-loader/examples/simple/');
+        stdout = run(path, [], false).stdout;
```

Comment:
```suggestion
        ({ stdout } = run(path, [], false));
```
What about the destructuring approach?

Reply:
I think its just personal preferences, I have no problem writing it like that.

Reply:
Should I change this @jamesgeorge007 ?

Reply:
I would've went ahead with object destructuring. But yeah, as you said it's just personal preference.
/cc @webpack/cli-team

Reply:
you can do it for consistency sake